src/vaults/mainnet.json

```diff
@@ -575,6 +575,11 @@
       "slug": "wasabi-shoney",
       "stakeTokenAddress": "0xd948212f077e552533158becbc1882c1b19c40fe"
     },
+    {
+  "beraRewardsVault": "0x39b08cCbdfD9B0D3C45F1FD02E9b57B44C88ed71",
+  "slug": "osbgt-ewbera-4",
+  "stakeTokenAddress": "0x0B77545A4EB1B995e79BF85e0ed8d6Ee288ca632"
```

Comment:
Please add the token information for `0x0B77545A4EB1B995e79BF85e0ed8d6Ee288ca632`